### ISSUE TITLE
Introduce a config strategy that involves only using DB updates

### ISSUE THREAD

**Issue by bkosborne** [acquia/blt#3254]
BLT uses the `drupal:config:import` command to perform configuration imports and database updates. There are several strategies supported:

* none
* core-only
* config-split
* features

If 'none' is selected, then nothing happens, not even `drush updb` is run. The problem is none of the other strategies apply to us, but we still need database updates to run.

In our use case, we're running a large multisite distro in which we do not use config management imports/exports, because each site is different enough where it's not practical. Even using config split is a bit too complicated at the moment. We do use features, but we don't use features to revert the configurations automatically. We're are manually reverting specific configurations we want using database update hooks.

So really, our config management strategy is something akin to 'manual'.

Would it be possible to add such a strategy that simply executes database updates?

**Comment by wu-edward** [comment]
Do you have an existing workaround for this for the time being? If not, you can add an annotated command hook https://blt.readthedocs.io/en/latest/extending-blt/#adding-a-custom-robo-hook to run drush database updates.

**Comment by bkosborne** [comment]
I worked around this by replacing the `drupal:config:import` with my own using the command replace annotation.

**Comment by danepowell** [comment]
Have you tried using `blt drupal:update` instead of `blt drupal:config:import`? That sounds exactly like what you are looking for: it runs db updates, then calls `blt drupal:config:import`. Architecturally, that seems a lot more appropriate than moving db updates into the configuration management namespace.

**Comment by bkosborne** [comment]
@danepowell  If I was invoking it directly, I would, but `blt drupal:config:import` is invoked as a sub-command of several other commands:

* drupal:install
* artifact:update:drupal:all-sites
* deploy:update
* drupal:update

**Comment by danepowell** [comment]
My point is that it's okay that `drupal:config:import` is called, as long as it's a subcommand of `drupal:update`, since `drupal:update` will run db updates regardless of your config strategy.

To your specific points:
- `drupal:install`: You wouldn't want to run db updates immediately after a drupal install anyway.
- `drupal:update`: This is the command I'm saying should be invoked.
- `artifact:update:drupal:all-sites` and `deploy:update`: These both invoke `drupal:update`, not config import directly: https://github.com/acquia/blt/blob/e09363be4aaf0b0fdb849b6ec5c895c4fa8b60d2/src/Robo/Commands/Deploy/DeployCommand.php#L641

In short, I don't see any reason why this shouldn't be working as-is, as long as you aren't manually calling `drupal:config:import` somewhere.

**Comment by danepowell** [comment]
Closing for now since there has been no response in over a week.
